src/main/java/ch/jalu/configme/beanmapper/MapperImpl.java

```diff
@@ -369,30 +379,21 @@ public MapperImpl(@NotNull BeanDescriptionFactory beanDescriptionFactory,
         if (!(value instanceof Map<?, ?>)) {
             return null;
         }
```

Comment:
This check is weird now if we have a bean instantiation that doesn't require any properties, resp. if the bean has default values everywhere?

Reply:
I think it still makes sense. An "empty bean" would be exported as an empty map. If it's not present, the default value will kick in. I don't see any case where we'd see something like an int and decide it makes more sense to try and map a bean instead of using the default value.